apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -21,13 +21,13 @@
 namespace Google.Cloud.Compute.V1.IntegrationTests
 {
     /// <summary>
-    /// Miscellaneous acceptance tests for the Compute API.
+    ///     Miscellaneous acceptance tests for the Compute API.
```

Comment:
Please unindent this. (It may well be ReSharper indenting by default, but it's harder to read like this.)

Reply:
reverted